pkg/integrations/v2/mssql/sql_exporter.go

```diff
@@ -0,0 +1,149 @@
+package snowflake_exporter
```

Comment:
I think this package name is wrong, right?

Reply:
Yep, good catch!